_howdoi/access-spark-webui.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+[id='access-spark-webui']
+= access the Apache Spark web interface?
+:page-layout: howdoi
+:page-menu_entry: How do I?
+
+Apache Spark provides an HTML-based web interface that displays information
+about the state of the processing cluster as well as applications that have
+been run on the cluster.
```

Comment:
possible change to: "the processing cluster and previously executed applications."

Reply:
otherwise lgtm